CAP/gap/OppositeCategory.gi

```diff
@@ -243,22 +254,30 @@ BindGlobal( "CAP_INTERNAL_INSTALL_OPPOSITE_ADDS_FROM_CATEGORY",
                                 "HorizontalPostCompose",
                                 "VerticalPreCompose",
                                 "VerticalPostCompose",
-                                "IdenticalTwoCell",
-                                "IsomorphismFromImageObjectToKernelOfCokernel" ] then
+                                "IdenticalTwoCell" ] then
             continue;
         fi;
         
-        if IsBound( current_entry.dual_operation ) then
-            dual_name := current_entry.dual_operation;
-        else
-            dual_name := current_recname;
+        ## Conservative
+        if not IsBound( current_entry.dual_operation ) then
+            continue;
         fi;
         
+        dual_name := current_entry.dual_operation;
+        
         if CurrentOperationWeight( category_weight_list, dual_name ) = infinity then
             continue;
         fi;
         
-        func := create_func( dual_name );
+        if IsBound( current_entry.dual_arguments_reversed ) then
+            
+            func := create_func( dual_name, true );
+            
+        else
+            
+            func := create_func( dual_name );
+            
+        fi;
```

Comment:
Should we add `false` as a default for `dual_arguments_reversed`? This would make the record entry more complete, and this if would disappear.